main/main.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
optional: it's nice when the folder name is a meaningful name that way by default the built binary has a good name. there's an informal/unofficial project layout that I tend to reference https://github.com/golang-standards/project-layout

Reply:
Thanks, fixed.